src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitStorage = new HashMap<>();
+
+    public static Map<String, Integer> getFruitStorage() {
+        return fruitStorage;
+    }
+
+    public static void addFruit(String fruit, int quantity) {
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public static void removeFruit(String fruit, int quantity) {
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) - quantity);
```

Comment:
When removing fruits, you should check if the resulting quantity is not negative. According to the checklist, a RuntimeException should be thrown if the balance is negative after a purchase operation.